rs/tests/consensus/subnet_recovery/common.rs

```diff
@@ -614,10 +630,9 @@ fn local_recovery(node: &IcNodeSnapshot, subnet_recovery: AppSubnetRecovery, log
         .unwrap_or_default();
     let pub_key = subnet_recovery.params.pub_key.unwrap();
     let pub_key = pub_key.trim();
-    let node_ip = node.get_ip_addr();
 
     let command = format!(
-        r#"/opt/ic/bin/ic-recovery \
+        r#"IC_ADMIN_BIN="/var/lib/admin/ic-admin" /opt/ic/bin/ic-recovery \
```

Comment:
nit: might be worth extracting this into a `const` for clarity (so that the varname can be searched which will lead to the `scp` command above)